config/settings/local.py

```diff
@@ -51,8 +51,12 @@
 if env("USE_DOCKER") == "yes":
     import socket
 
-    hostname, _, ips = socket.gethostbyname_ex(socket.gethostname())
-    INTERNAL_IPS += [".".join(ip.split(".")[:-1] + ["1"]) for ip in ips]
+    try:
+        _, _, ips = socket.gethostbyname_ex("node")
+        INTERNAL_IPS.extend(ips)
+    except socket.gaierror:
+        # The node container isn't started (yet?)
```

Comment:
I think it would make sense to at least log something here, otherwise it could cause confusion if things are not working later

Reply:
Actually, I shouldn’t have commented it. 🙏